apps/web/src/components/NotificationList/NotificationListPage.tsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { useQuery } from "urql";
+import { useSearchParams } from "react-router-dom";
+
+import { graphql } from "@gc-digital-talent/graphql";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { Link, Pending, Well } from "@gc-digital-talent/ui";
+
+import NotificationItem from "./NotificationItem";
+
+const Notifications_Query = graphql(/* GraphQL */ `
+  query Notifications(
+    $where: NotificationFilterInput
+    $first: Int
+    $page: Int
+  ) {
+    notifications(where: $where, first: $first, page: $page) {
+      paginatorInfo {
+        hasMorePages
+      }
+      data {
+        id
+        ...NotificationItem
+      }
+    }
+  }
+`);
+
+const PER_PAGE = 10;
+
+interface NotificationPageProps {
+  page: number;
+  onlyUnread?: boolean;
+  isLastPage?: boolean;
+}
+
+const NotificationListPage = ({
+  page,
+  onlyUnread,
+  isLastPage,
+}: NotificationPageProps) => {
+  const intl = useIntl();
+  const [searchParams] = useSearchParams();
+  searchParams.set("page", `${page + 1}`);
+  const [{ data, fetching, error }] = useQuery({
+    query: Notifications_Query,
+    variables: {
+      first: PER_PAGE,
+      page,
+      where: {
+        onlyUnread,
+      },
+    },
+  });
+
+  const notifications = unpackMaybes(data?.notifications?.data);
+
+  if (notifications.length === 0 && page === 1 && !fetching) {
+    return (
+      <Well>
+        <p data-h2-font-weight="base(700)" data-h2-margin-bottom="base(x1)">
+          {intl.formatMessage({
+            defaultMessage: "There aren't any notifications here yet.",
```

Comment:
@substrae All the copy here about the future strikes me as strange.  There aren't any notifications here probably because I just marked them as read or deleted them.  Not because I haven't received any yet.

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/224365e2-df7d-4899-a875-7171f8f725c4)

Hopefully the amount of time for which a user will have never received a notification will be pretty short, right?

Reply:
🤔 won't you only ever receive notifications in a future sense though? Just because you dismissed them doesn't mean the statements there aren't true. Happy to update it if y'all have better suggestions though!

Reply:
I'd be happy if we just dropped the word "yet" from the title.  The rest would be fine, I think.

Reply:
@petertgiles works for me!